### PR TITLE
📰 Added Google Messages Presence

### PR DESCRIPTION
**Pull request by peaillaud**
Here are some screenshots of the Presence for Google Messages Web:
Screenshot_1
Screenshot_2
Screenshot_3

Thank you!

### CODE REVIEW

**Comment by peaillaud** [comment]
whoaaa finally

File: websites/G/Google Messages/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "name": "Paulé",
+        "id": "376812375795302402"
+    },
+    "service": "Google Messages",
+    "description": {
+        "en": "Google Messages is a simple, helpful messaging app that keeps you connected with the people who matter most. Text anyone from anywhere across devices."
+    },
+    "url": "messages.google.com",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/sWaECpP.png",
```
Comment (Bas950) [inline websites/G/Google Messages/dist/metadata.json:13]: logo must be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/Google Messages/presence.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        smallImageText: "Google"
+    },
+    path = document.location.pathname.toLowerCase(),
+    showcon = await presence.getSetting("showContact");
+    // Home Page
+    if(path === "/" || path.includes("/intl/")) {
+        presenceData.largeImageKey = "icon";
+        presenceData.details = "Home page";
+        presenceData.startTimestamp = tmb;
+    }
```
Comment (Bas950) [inline websites/G/Google Messages/presence.ts:18]: instead of setting it in every if, just set it at the start

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/Google Messages/presence.ts [hunk 3]
```diff
@@ -6,26 +6,24 @@ tmb = Math.floor(Date.now() / 1000);
 presence.on("UpdateData", async () => {
     const presenceData: PresenceData = {
         smallImageKey: "google",
-        smallImageText: "Google"
+        smallImageText: "Google",
+        presenceData.startTimestamp = tmb;
```
Comment (Bas950) [inline websites/G/Google Messages/presence.ts:10]: yea this doesnt work, just startTimestamp:

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/Google Messages/presence.ts (hunk 3 above)
Comment (ririxidev) [reply websites/G/Google Messages/presence.ts:10]: KEKW

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**